cardano-testnet/test/Spec/ShutdownOnSlotSynced.hs

```diff
@@ -0,0 +1,67 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE RecordWildCards #-}
```

Comment:
Can we use `NamedFieldPuns` instead? I find `RecordWildCards` too opaque.

Reply:
I'll just get rid of it and use projection functions